### PR TITLE
Fixed: false positives for hyphenated function names in `function-calc-no-unspaced-operator`.

### PR DESCRIPTION
**Pull request by evilebottnawi**
> Which issue, if any, is this issue related to?

#1878 

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

### CODE REVIEW

File: src/rules/function-calc-no-unspaced-operator/index.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ export default function (actual) {
 }
 
 function blurVariables(source) {
-  return source.replace(/[\$@][^\)\s]+/g, "0")
+  return source.replace(/([\$@][^\)\s]+|#{.+?})/g, "0")
```
Comment (davidtheclark) [inline src/rules/function-calc-no-unspaced-operator/index.js]: There is no need for the outer parentheses, is there?

File: src/rules/function-calc-no-unspaced-operator/__tests__/index.js [hunk 2]
```diff
@@ -77,6 +77,12 @@ testRule(rule, {
     code: "a { top: calc(2rem + @fh+d*sf-as); }",
     description: "Less variable with symbols",
   }, {
+    code: "a { top: calc(100% - #{$foo}); }",
+    description: "Scss interpolation",
+  }, {
+    code: "a { top: calc(100% - #{map-get($container-max-widths, xl)}); }",
+    description: "Scss interpolation with function",
+  }, {
```
Comment (davidtheclark) [inline src/rules/function-calc-no-unspaced-operator/__tests__/index.js]: Can we have a test showing that it errors with `a { top: calc(100%- #{$foo}); }` or `a { top: calc(100% -#{$foo}); }`?

File: src/rules/function-calc-no-unspaced-operator/index.js (hunk 1 above)
Comment (evilebottnawi) [inline src/rules/function-calc-no-unspaced-operator/index.js]: @davidtheclark yep, habit :smile:

**Comment by davidtheclark** [comment]
Added to changelog:

> -   Fixed: false positives for SCSS interpolation in `function-calc-no-unspaced-operator`.

**Pull request merged**